### DISCUSSION TITLE
"Outpost has inconsistent dimensions"

### DISCUSSION THREAD

**Discussion by ibalshammari** [Open-Systems-Pharmacology/Forum#1013] [Questions & Problems] [answered]
Hello,
I tried to do optimization from “Parameter Identification”, but I received an error message which is “outpost has inconsistent dimensions”!
I am wondering if anyone can help me to solve this issue. Thanks!

**Comment by msevestre** [comment]
@ibalshammari 
Where are you getting the error message?
Can you show us a screenshot?

**Reply by ibalshammari** [reply]
![Screenshot 2021-11-08 184254](https://user-images.githubusercontent.com/93001986/140846586-1f76ffd3-37c5-4f22-b34f-e9d53578be46.png)

**Comment by msevestre** [comment]
Can you also show the screen with the output selection (first tab on the parameter identification panel). You probably have different dimensions between your selected output and observed data

**Reply by ibalshammari** [reply]
![Screenshot 2021-11-08 194918](https://user-images.githubusercontent.com/93001986/140852871-6f80d214-66e5-45f2-8718-df2a444225f4.png)

**Comment by msevestre** [comment, marked as answer]
Look at the first line. There is a white cross on red background indicating an error. Go over with your mouse for more details. But in short, you are mixing entities of different dimensions. Either change the dimensions of your observed data or select another output

**Reply by ibalshammari** [reply]
Yes, I see that. I'll try to change the dimensions of my observed data to fix it. 
Thanks a lot, Michael!
